src/EditorFeatures/Test2/Simplification/TypeNameSimplifierTest.vb

```diff
@@ -1517,7 +1602,7 @@ class Program
 
         var d = new D();
 
-        foreach (var item in new List&lt;int&gt;()) { }
+        foreach (int item in new List&lt;int&gt;()) { }
```

Comment:
this is appropriate.  there is no "using System.Collection.Generic", so it was not appropriate for that 'int' to be converted to 'var'.  This changed when i made the simplifier actually use the real "can use var" analysis code.